core-web/apps/dotcms-ui/src/app/portlets/dot-containers/dot-container-create/dot-loop-editor/dot-loop-editor.component.spec.ts

```diff
@@ -88,16 +88,16 @@ fdescribe('DotLoopEditorComponent', () => {
         fixture.detectChanges();
     });
 
-    fit('should create', () => {
+    it('should create', () => {
         expect(component).toBeTruthy();
     });
 
-    fit('should show pre_loop', () => {
+    it('should show pre_loop', () => {
         const label = de.query(By.css('[data-testId="label"]')).nativeElement;
         expect(label.innerText).toBe('Pre-loop');
     });
 
-    fit('should show pre_loop', fakeAsync(() => {
+    it('should show pre_loop', fakeAsync(() => {
         component.label = 'post_loop';
         fixture.detectChanges();
         tick();
```

Comment:
Why this need tick and fake async?

Reply:
I removed the `fakeAsync` but when I run the test previously it's failing but now it's working fine.